cpp/benchmarks/CMakeLists.txt

```diff
@@ -39,8 +39,9 @@ include_directories("${CMAKE_BINARY_DIR}/include"
                     "${CMAKE_SOURCE_DIR}"
                     "${CMAKE_SOURCE_DIR}/src"
                     "${CMAKE_SOURCE_DIR}/thirdparty/dlpack/include"
-                    "${GTEST_INCLUDE_DIR}"
-                    "${GBENCH_INCLUDE_DIR}"
+                    "${gtest_SOURCE_DIR}/include"
+                    "${gmock_SOURCE_DIR}/include"
+                    "${benchmark_SOURCE_DIR}/include"
```

Comment:
These are all unsafe to use if they're found via a binary install like apt or conda since we won't have the source.